compose/README.md

```diff
@@ -114,9 +114,9 @@ Now you can use `dbeaver-te` command to start manager.
 
 For detailed instructions on how to use the script manager, refer to [manager doucmentation](../manager/README.md).
 
-#### Version update procedure to 24.1.0
+#### Version update procedure to 24.2.0
```

Comment:
Here should be 24.1.0
Users may have problems only when updating from 23.3.0 to 24.0.1 or higher